app/test_split_integer.py

```diff
@@ -1,21 +1,39 @@
 from app.split_integer import split_integer
 
 
```

Comment:
Add an error message for each test. It's good when the user understands what the problem is